app/views/comments/_liquid.html.erb

```diff
@@ -19,6 +19,14 @@
     <%= render "comments/comment_date", decorated_comment: comment.decorate %>
   </div>
   <div class="body">
+    <% if comment.decorate.display_edited? %>
+      <div class="comment-edited-notice comment-date">
+        Edited
+        <time datetime="<%= comment.decorate.edited_timestamp %>">
+          <%= comment.decorate.edited_at %>
+        </time>
+      </div>
+    <% end %>
```

Comment:
since this is the same as the other maybe it can be put in a partial to avoid possible discrepancies over time